src/structlog/stdlib.py

```diff
@@ -499,7 +499,7 @@ def format(self, record):
         # Make a shallow copy of the record to let other handlers/formatters
         # process the original one
         record = logging.makeLogRecord(record.__dict__)
-        try:
+        if hasattr(record, "_logger") and hasattr(record, "_name"):
```

Comment:
Please don't use [hasattr()](https://hynek.me/articles/hasattr/). I would suggest a global symbol `_SENTINEL = object()` and check using `is`.

Reply:
Ah yes! Great article.